applications/crossbar/src/modules_v2/cb_devices_v2.erl

```diff
@@ -838,6 +837,7 @@ set_mobile_public_fields(Normalized, Context) ->
     Options = [{'auth_by', AuthAccountId}
               ,{'dry_run', not cb_context:accepting_charges(Context)}
               ,{'public_fields', PublicFields}
+              ,{'state', ?NUMBER_STATE_IN_SERVICE}
```

Comment:
`move` already sets state to `in_service`. Setting this here initializes `Normalized` with a possibly wrong state, before going through transition checks.